docs/docsite/rst/conf.py

```diff
@@ -395,19 +395,19 @@
     'python3': ('https://docs.python.org/3/', None, None),
     'jinja2': ('http://jinja.palletsprojects.com/', None, None),
     'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', None, None),
-    'ansible_8': ('https://docs.ansible.com/ansible/8/', None, None),
+    'ansible_9': ('https://docs.ansible.com/ansible/9/', None, None),
 } if tags.has('all') else {
     'python': ('https://docs.python.org/2/', None, None),
     'python3': ('https://docs.python.org/3/', None, None),
     'jinja2': ('http://jinja.palletsprojects.com/', None, None),
     'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', None, None),
-    'ansible_8': ('https://docs.ansible.com/ansible/8/', None, None),
+    'ansible_9': ('https://docs.ansible.com/ansible/8/', None, None),
```

Comment:
```suggestion
    'ansible_9': ('https://docs.ansible.com/ansible/9/', None, None),
```

Reply:
An unrelated note: Those separate values `None, None` should be grouped together. Some entries have it correctly, as a tuple `(None, None)`, but many don't. I didn't want to fix it while melting the configs together so it got postponed. Still, this is something that needs to be done, eventually (in a separate PR).

Reply:
fixed in https://github.com/ansible/ansible-documentation/pull/880  thanks!